scripts/lib/CIME/Servers/wget.py

```diff
@@ -15,9 +15,9 @@ def __init__(self, address, user='', passwd=''):
             self._args += "--password {} ".format(passwd)
         self._server_loc = address
 
-        err = run_cmd("wget {} --spider {}".format(self._args, address))[0]
-        expect(err == 0,"Could not connect to repo '{0}'\nThis is most likely either a proxy, or network issue .")
-
+        cmd = "wget {} --no-check-certificate --spider {}".format(self._args, address)
```

Comment:
Is this addition wise?  Maybe it should be an option in the xml that is implemented on a per server basis?